Source/Particles/MultiParticleContainer.cpp

```diff
@@ -566,9 +567,9 @@ MultiParticleContainer::DepositCharge (
     const amrex::Real relative_time)
 {
     // Reset the rho array
-    for (int lev = 0; lev < rho.size(); ++lev)
+    for (auto& rho_lv : rho)
     {
-        rho[lev]->setVal(0.0_rt);
+        rho_lv->setVal(0.0_rt);
```

Comment:
Could we call this `rho_lev` instead of `rho_lv`, so that it is as self-explanatory as possible?